docs/dev-proc/smbios-rules.md

```diff
@@ -0,0 +1,69 @@
+# Dasharo Product Guidelines: SMBIOS Information
+
+## Introduction
+
+This document describes the guidelines for filling in the SMBIOS fields in
+devices supported by Dasharo. It is meant to be used primarily as a reference
+for Dasharo developers.
+
+## BIOS Information (Type 0)
+
+### Vendor field
+
+The vendor field is always set as follows:
+
+```
+3mdeb
+```
+
+*for firmware first released before January 13 2022:*
+
+```
+3mdeb Embedded Systems Consulting
+```
+
+### Version field
+
+The BIOS version string is defined as follows:
+
+```bash
+Dasharo ([major_framework]+[minor_framework]) v[version]
+```
+
+- `major_framework` refers to the primary framework in which the firmware is
+  developed, e.g. `coreboot` or `edk2`.
+
+- `minor_framework` is the secondary framework, a supporting component for the
+  primary framework, e.g. a coreboot payload. May not always be used, e.g. if
+  `edk2` is used as the primary framework.
+
+- `version` means version according to official [Dasharo Versioning](../versioning) documentation.
+
+*For [TianoCore UEFIPayloadPkg](https://github.com/Dasharo/edk2/tree/workstation/master/UefiPayloadPkg),
+we use the simpler and more recognizable UEFI name.*
+
+Examples:
+
+- `Dasharo (coreboot+UEFI) v1.0.0` - for coreboot-based builds with TianoCore UEFIPayload
+- `Dasharo (coreboot+SeaBIOS) v1.0.0` - for coreboot-based builds with SeaBIOS payload
+- `Dasharo (coreboot+Heads) v1.0.0` - for coreboot-based builds with Heads payload
+- `Dasharo (edk2) v1.0.0` - for plain edk2-based builds
```

Comment:
Now I wonder if this shouldn't be UEFI instead of edk2. @macpijan @miczyg1 @krystian-hebel ?

Reply:
We can also have an older edk2 corebootPayloadPkg. The variety here makes it complex. I don't think anything else than edk2 is worth integrating. We prioritize functionality that addresses customer requirements and only edk2 is able to fulfill this need. I would keep UEFI to simplify things. EDK2 is a set of packages and libraries used to create a UEFI firmware. coreboot is a firmware but an incomplete one. so coreboot+STH is meaningful, but edk2 alone isn't. There would have to be edk2 MinPlatform if anything.

tl;dr
I vote for keeping UEFI

Reply:
@krystian-hebel we already discussed that topic about using generic UEFI name and we agree that UEFI is better marketing term known to everyone. edk2 is unknown term and it will cause confusion. What we can say it mean in this place is UEFI compatibility, so firmware expose those interfaces - we say nothing about implementation we using, because as @miczyg1 said it complicates whole explanation.

Reply:
+1 for UEFI from me in light of the above arguments. We can worry for different UEFI payloads when (if?) we will have such cases. This name should be rather easy to understand. We explain in SBOM which exact projects were used to build the firmware.

Reply:
@mkopec please replace edk2 with UEFI.

Reply:
Replaced